src/app/public/modules/ag-grid/ag-grid.service.spec.ts

```diff
@@ -266,6 +271,58 @@ describe('SkyAgGridService', () => {
     });
   });
 
+  describe('onKeyPress', () => {
+    let onKeyPressFunction: Function;
+    let gridApi: GridApi;
+    let keypress: CellKeyPressEvent;
+    let event: KeyboardEvent;
+
+    beforeEach(() => {
+      onKeyPressFunction = defaultGridOptions.onCellKeyPress;
```

Comment:
These tests validate the logic inside `onCellKeyPress()`, but they don't validate that that function (or a function that calls that function) is specified for the `onCellKeyPress` property on the grid options object. I'd change these tests to call the `getGridOptions()` method, then grab the function from the resulting object's `onCellKeyPress` property.

Reply:
`defaultGridOptions` is set by calling `getGridOptions` in the `beforeEach()`

Reply:
Sorry, I completely misread this code. You did exactly what I suggested in your code.